src/web/components/ArticleBody.tsx

```diff
@@ -91,6 +91,9 @@ export const ArticleBody = ({
 	}
 	return (
 		<div
+			// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+			tabIndex={0}
```

Comment:
It’s unclear why we introduced this rule. There seems to be no benefit in making the main content tabbable.

In most other places, we explicitely use `tabIndex={-1}`, which “ensures that focus continues throught the page and deos not stay on the skip link itself”.